docs/creating_integration_manifest.md

```diff
@@ -414,9 +414,7 @@ For example:
 
 The [Integration Quality Scale](https://www.home-assistant.io/docs/quality_scale/) scores an integration on the code quality and user experience. Each level of the quality scale consists of a list of requirements. If an integration matches all requirements, it's considered to have reached that level.
 
-When your integration has no score, then don't add it to the manifest of your integration. However, be sure to look at the [Integration Quality Scale](https://www.home-assistant.io/docs/quality_scale/) list of requirements. It helps to improve the code and user experience tremendously.
-
-We highly recommend getting your integration scored.
+New integrations are required to fulfill at least the bronze tier so be sure to look at the [Integration Quality Scale](https://www.home-assistant.io/docs/quality_scale/) list of requirements. It helps to improve the code and user experience tremendously.
```

Comment:
I suggest we now link this to the developer documentation, not the end-user documentation version.

../Frenck